eslint-common.json

```diff
@@ -9,6 +9,7 @@
   "dot-location": ["property"],
   "eol-last": ["always"],
   "eqeqeq": ["always", {"null": "never"}],
+  "for-direction": [],
```

Comment:
Just out of curiosity, why disable this rule in `common`, and enable it in `es3` and `es6`?